src/test/java/com/titarenko/rest/HttpControllerTest_withoutMocks.java

```diff
@@ -12,68 +13,207 @@
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
-import java.util.Collections;
-import java.util.List;
+import java.time.LocalDate;
+import java.util.*;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class HttpControllerTest_withoutMocks {
 
     private static final HttpClient HTTP_CLIENT = HttpClient.newHttpClient();
     public static final JsonParser JSON_PARSER = new JsonParser();
-    private static Employee vasil = new Employee.Builder().withName("Vasil").withSalary(150).build();
+    private static List<Employee> database;
+    private static Employee vasil = new Employee.Builder()
+            .withId(1)
+            .withName("Vasil")
```

Comment:
Create a `src/test/java/com/titarenko/UnitTestParent.java`:
```
public class UnitTestParent {
    public LinkedList<Employee> database;   
    public Employee vasil = ...
    public Employee petr = ...
    public Employee stepa = ...
    public EmployeeDao dao = new EmployeeDao() {
        ....
    };
}
```
Extend `HttpControllerTest_withoutMocks` and `EmployeeServiceImplTest_withoutMocks` from `UnitTestParent`.